frontend/src/app/search/results-sort/search-sorting.component.spec.ts

```diff
@@ -1,13 +1,13 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-import { mockCorpus } from '../../mock-data/corpus';
-import { commonTestBed } from '../common-test-bed';
-import { QueryModel } from '../models';
+import { mockCorpus } from '../../../mock-data/corpus';
+import { commonTestBed } from '../../common-test-bed';
+import { QueryModel } from '../../models';
```

Comment:
The imports for models and services would benefit from path mapping in `tsconfig.json`